### DISCUSSION TITLE
how to bundle default plugins on code-server

### DISCUSSION THREAD

**Discussion by SukiiL** [coder/code-server#5267] [Q&A]
how to bundle default extensions on code-server so that each user could have these extensions  when they use code-server for the first time?

**Comment by jsjoeio** [comment]
One way could be to have some sort of script which installs them in the environment. [This](https://coder.com/docs/coder/latest/workspaces/vs-code-extensions#adding-extensions-to-custom-images) is how we recommend it in our product.
